pkg/instancetype/instancetype_test.go

```diff
@@ -2228,6 +2228,45 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*vmi.Spec.Domain.Firmware.Bootloader.EFI.Persistent).To(BeTrue())
 				Expect(vmi.Spec.Domain.Firmware.Bootloader.EFI.SecureBoot).To(BeNil())
 			})
+
+			It("should not overwrite EFI when using PreferredEfi - bug #12985", func() {
```

Comment:
nit: Maybe several `It`s can be merged in a `DescribeTable`.
For a followup maybe, not here :)

Reply:
I'd just end up duplicating the assertions in the table so meh. We are starting to heavily refactor these tests in 1.4 so I'll take another look then thanks.